command/sub8_launch/config/orange_rectangle_finder.yaml

```diff
@@ -0,0 +1,17 @@
+debug_ros: True
```

Comment:
Most of this is self explanatory, but comments briefly saying what each does would be good. Also some of them could use unit definitions too